templates/shared_services/gitea/Dockerfile

```
@@ -1,3 +1,6 @@
-FROM gitea/gitea:latest
-RUN wget -O /usr/local/share/ca-certificates/mysql.crt.pem https://www.digicert.com/CACerts/BaltimoreCyberTrustRoot.crt.pem && update-ca-certificates
+ARG GITEA_TAG=1.15
+ARG CERTIFICATE_URL=https://www.digicert.com/CACerts/BaltimoreCyberTrustRoot.crt.pem
+
+FROM gitea/gitea:${GITEA_TAG}
+RUN wget -O /usr/local/share/ca-certificates/mysql.crt.pem ${MYSQL_CERTIFICATE_URL} && update-ca-certificates
```

```suggestion
RUN wget -O /usr/local/share/ca-certificates/mysql.crt.pem ${CERTIFICATE_URL} && update-ca-certificates
```
I think there is a typo here.